common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +97,876 @@ innovation_taming_the_skies = {
 
 	potential = {
 		is_giant_culture_trigger = no
+		NOT = {
+			has_aerial_maa_trigger = yes
+		}
 	}
 
 	unlock_maa = aerial_unit
 
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
+### MAA TECHS
+
+####Tribal
+innovation_archmage = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @archmage
+
+	potential = {
+		AND = {
+			OR = {
+				this.culture_group = culture_group:human_group
+				this = culture:blood_elf
+				this = culture:high_elf
+			}
+			NOT = {
+				this = culture:pirate_culture
+				this = culture:wastewander	
+			}
+		}
+	}
+
+	unlock_maa = archmage
+
+	flag = global_maa
+}
+
+
+innovation_blademaster = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @horde_special
+
+	potential = {
+		this = culture:burning_blade
+	}
+
+	unlock_maa = blademaster
+
+	flag = global_maa
+}
+
+innovation_buccaneer = {
```

Comment:
I would give both pirate units jungle bonus since the South Seas are their main region.

Reply:
👌